galite-core/src/main/kotlin/org/kopi/galite/report/PExport.kt

```diff
@@ -35,9 +35,7 @@ abstract class PExport(val table: UTable,
     for (j in 0 until model.getAccessibleColumnCount()) {
       val visibleColumn: Int = table.convertColumnIndexToModel(j)
       val column: VReportColumn = model.getAccessibleColumn(visibleColumn)
-      if (column.isVisible() && column.isFolded() == false // if we have a new page for each group, we do not use the first
-              // visible column
-              && (!printConfig.groupFormfeed || j != firstVisibleColumn)) {
+      if (column.visible && !column.folded && (!printConfig.groupFormfeed || j != firstVisibleColumn)) { // if we have a new page for each group, we do not use the first visible column
```

Comment:
if (column.visible && !column.folded// if we have a new page for each group, we do not use the first
              // visible column
 && (!printConfig.groupFormfeed || j != firstVisibleColumn)) { // if we have a new page for each group, we do not use the first visible column